src/styles/Search.css

```
@@ -0,0 +1,34 @@
+.search__input {
+  border-radius: 11px;
+  font-size: 10px;
+  padding: 5px 10px;
+  border: 1px solid black;
+  height: 100%;
+}
+
+.search__button {
+  font-size: 10px;
+  color: white;
+  padding: 5px 10px;
+  border-radius: 7px;
+  background-color: rgb(239, 75, 75);
+  border: 1px solid rgba(0, 0, 0, 0);
+}
+
+@media screen and (min-width: 768px) {
+  .search__input {
+    font-size: 20px;
+  }
+  .search__button {
+    font-size: 20px;
+  }
```

Could be consolidated:
```suggestion
  .search__input,
  .search__button {
    font-size: 20px;
  }
```